thirdparty/tinymce/tiny_mce_gzip.php

```diff
@@ -19,7 +19,7 @@
 	// Default settings
 	$tinyMCECompressor = new TinyMCE_Compressor(array(
 		// CUSTOM SilverStripe
-		'cache_dir' => TEMP_FOLDER
+		'cache_dir' => TEMP_FOLDER.'/tinymce-cache'

```

Comment:
ooh, I hadn't realised it didn't get written :( I'll make the relevant changes soon